--- raqm-0.2.0/src/raqm.c.orig	2016-08-25 14:12:55.000000000 +0200
+++ raqm-0.2.0/src/raqm.c	2017-05-10 21:52:39.004931212 +0200
@@ -1541,7 +1541,7 @@
 
   if (rq->flags & RAQM_FLAG_UTF8)
     *index = _raqm_u32_to_u8_index (rq, *index);
-  RAQM_TEST ("The position is %d at index %ld\n",*x ,*index);
+  RAQM_TEST ("The position is %d at index %zd\n",*x ,*index);
   return true;
 }
 
@@ -1625,7 +1625,7 @@
         }
         if (_raqm_allowed_grapheme_boundary (rq->text[*index],rq->text[*index + 1]))
         {
-          RAQM_TEST ("The start-index is %ld  at position %d \n", *index, x);
+          RAQM_TEST ("The start-index is %zd  at position %d \n", *index, x);
             return true;
         }
 
@@ -1639,7 +1639,7 @@
           }
           *index += 1;
         }
-        RAQM_TEST ("The start-index is %ld  at position %d \n", *index, x);
+        RAQM_TEST ("The start-index is %zd  at position %d \n", *index, x);
         return true;
       }
       else
@@ -1653,7 +1653,7 @@
   else
     *index = rq->text_len;
 
-  RAQM_TEST ("The start-index is %ld  at position %d \n", *index, x);
+  RAQM_TEST ("The start-index is %zd  at position %d \n", *index, x);
 
   return true;
 }
